### PR TITLE
chore: const variable in header

### PR DESCRIPTION
**Pull request by Decodetalkers**
const variable should be marked inline in header

Log: variable in header marked inline

### CODE REVIEW

File: src/plugin-update/operation/common.h [hunk 1]
```diff
@@ -14,23 +14,24 @@
...
-const bool IsServerSystem = (DSysInfo::UosServer == UosType);          // 是否是服务器版
-const bool IsCommunitySystem = (DSysInfo::UosCommunity == UosEdition); // 是否是社区版
-const bool IsProfessionalSystem = (DSysInfo::UosProfessional == UosEdition); // 是否是专业版
-const bool IsHomeSystem = (DSysInfo::UosHome == UosEdition);                 // 是否是个人版
-const bool IsEducationSystem = (DSysInfo::UosEducation == UosEdition);       // 是否是教育版
-const bool IsDeepinDesktop = (DSysInfo::DeepinDesktop == DSysInfo::deepinType()); // 是否是Deepin桌面
-
-const QString TestingChannelPackage = QStringLiteral("deepin-unstable-source");
-const QString ServiceLink = QStringLiteral("https://insider.deepin.org");
+inline const double Epsion = 1e-6;
```
Comment (cheese) [inline src/plugin-update/operation/common.h:17]: `inline constexpr` is even better.

File: src/frame/utils.h [hunk 2]
```diff
@@ -22,6 +22,6 @@ QByteArray getFileMd5(const QString &filePath);
 ModuleObject *GetModuleByUrl(ModuleObject *const root, const QString &url);
 QString GetUrlByModule(ModuleObject *const module);
 
-const QString PLUGIN_DIRECTORY = QStringLiteral(DefaultModuleDirectory);
-const QString OLD_PLUGIN_DIRECTORY = QStringLiteral("/usr/lib/dde-control-center/modules/");
+inline const QString PLUGIN_DIRECTORY = QStringLiteral(DefaultModuleDirectory);
```
Comment (cheese) [inline src/frame/utils.h:25]: Don't define global variables from other variables.

File: src/plugin-update/operation/common.h [hunk 3]
```diff
@@ -14,23 +14,24 @@
...
+inline const QString SystemUpdateType = QStringLiteral("system_upgrade");
+inline const QString AppStoreUpdateType = QStringLiteral("appstore_upgrade");
+inline const QString SecurityUpdateType = QStringLiteral("security_upgrade");
+inline const QString UnknownUpdateType = QStringLiteral("unknown_upgrade");
+
+inline const DSysInfo::UosType UosType = DSysInfo::uosType();
+inline const DSysInfo::UosEdition UosEdition = DSysInfo::uosEditionType();
+inline const bool IsServerSystem = (DSysInfo::UosServer == UosType); // 是否是服务器版
```
Comment (cheese) [inline src/plugin-update/operation/common.h:25]: Don't define global variables from other variables.

File: src/frame/utils.h (hunk 2 above)
Comment (Decodetalkers) [reply src/frame/utils.h:25]: DefaultModuleDirectory is a marco by cmake

**Review by BLumia — APPROVED** [review]

**Pull request merged**